crates/sui-indexer/src/handlers/committer.rs

```diff
@@ -176,6 +178,17 @@ async fn commit_checkpoints<S>(
             );
         })
         .expect("Persisting data into DB should not fail.");
+
+    if is_epoch_end {
+        // The epoch has advanced so we update the configs for the new protocol version, if it has changed.
+        let chain_id = state
+            .get_chain_identifier()
+            .await
+            .expect("Failed to get chain identifier")
+            .expect("Chain identifier should have been indexed at this point");
+        let _ = state.persist_protocol_configs_and_feature_flags(chain_id);
+    }
```

Comment:
should we do this before we `persist_checkpoints`, maybe right after `state.advance_epoch` under `if let Some(epoch_data) = epoch`?

Reply:
We actually have to do this after `persist_checkpoints`, in the event where the first epoch is committed before the first checkpoints. In that scenario, chain identifier doesn't exist yet in the db when we advance epoch.